### PR TITLE
os/bluestore: some minor fixes/improvements for allocator's stats inquiries

### PR DESCRIPTION
**Pull request by ifed01**
Signed-off-by: Igor Fedotov <ifedotov@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-10-21_07:01:44-rados-wip-kefu-testing-2020-10-21-1144-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-10-22_03:25:39-rados-wip-kefu-testing-2020-10-21-2116-distro-basic-smithi/

File: src/os/bluestore/bluestore_tool.cc [hunk 1]
```diff
@@ -251,7 +251,7 @@ int main(int argc, char **argv)
     ("deep", po::value<bool>(&fsck_deep), "deep fsck (read all data)")
     ("key,k", po::value<string>(&key), "label metadata key name")
     ("value,v", po::value<string>(&value), "label metadata value")
-    ("allocator", po::value<vector<string>>(&allocs_name), "allocator to inspect: 'block'/'bluefs-wal'/'bluefs-db'/'bluefs-slow'")
+    ("allocator", po::value<vector<string>>(&allocs_name), "allocator to inspect: 'block'/'bluefs-wal'/'bluefs-db'")
```
Comment (tchaikov) [inline src/os/bluestore/bluestore_tool.cc:254]: i don't quite understand this.. but we still handles bluefs-slow when serving "bluefs files list" and in `BlueFS::mkfs()`, which in turn calls `BlueFS::_init_alloc()`.

**Comment by yuriw** [comment]
[wip-yuri-testing-2020-10-26-0817](https://trello.com/c/1Fh3DURx/1047-wip-yuri-testing-2020-10-26-0817)

File: src/os/bluestore/bluestore_tool.cc (hunk 1 above)
Comment (aclamk) [reply src/os/bluestore/bluestore_tool.cc:254]: @tchaikov
We used to have 4 allocators: 
a) 1 for BlueStore block data, 
b) 1 for BlueFS db, 
c) 1 for BlueFs wal, 
d) 1 for BlueFS on block.
When BlueFS was low on space it requested allocation from a) and transferred this space to d) as "new space". 

Now we got rid of d) and BlueFS is allocating space from a) directly.

So, we got rid of 1 allocator, but reporting of disk space used by BlueFS files remained unchanged.

**Comment by ifed01** [comment]
@aclamk - mind reviewing?

**Comment by tchaikov** [comment]
@aclamk  ping?

**Review by aclamk — APPROVED** [review]

**Pull request merged**